src/nunit-gui/Views/MainForm.cs

```diff
@@ -144,5 +144,10 @@ private void statusBarToolStripMenuItem_CheckedChanged(object sender, System.Eve
         {
             statusBarView.Visible = statusBarToolStripMenuItem.Checked;
         }
+
+        protected override void OnDragEnter(DragEventArgs drgevent){
+            if(drgevent.Data.GetDataPresent(DataFormats.FileDrop))
+                drgevent.Effect = DragDropEffects.Copy;
+        }
```

Comment:
My understanding is that this is merely the visual effect to tell the user that a drop is possible. Was the actual drop already working?